charts/octopus-deploy/tests/statefulset_test.yaml

```diff
@@ -71,4 +71,22 @@ tests:
           content: 
             name: "ACCEPT_EULA"
             value: "N"
+        documentIndex: 0
+  
+  - it: proxy is configurable
+    values:
+      - ./values/required.yaml
+      - ./values/proxy.yaml
```

Comment:
Can we make sure we test all possible inputs, especially if we end up using an array for the excluded hostnames